pytorch_lightning/trainer/data_loading.py

```diff
@@ -474,11 +477,16 @@ def request_dataloader(self, model: 'pl.LightningModule', stage: str) -> DataLoa
 
     def _flatten_dl_only(self, dataloaders):
         # handles user error when they return:
-        # return dl1, dl2  vs  return (dl1, dl2)
-        if isinstance(dataloaders, tuple):
-            all_dls = [isinstance(x, Iterable) for x in dataloaders]
-            all_dls = all(all_dls)
-            if all_dls:
-                dataloaders = list(dataloaders)
-
+        # `return dl1, dl2` vs `return (dl1, dl2)`
```

Comment:
anyone know what the difference is here?
what is this comment trying to say?

is the user error they are talking about here this?

`return (dl1, dl2), `

Reply:
Good question, I am not sure either.

Reply:
Checking this out in https://github.com/PyTorchLightning/pytorch-lightning/pull/8439